packages/dashboard-core-plugins/src/panels/IrisGridPanel.tsx

```diff
@@ -423,11 +423,6 @@ export class IrisGridPanel extends PureComponent<
         return null;
       }
 
-      // TODO #2093: Find a better way to handle deprecated panel prop
-      const deprecatedProps = {
-        panel: this,
-      };
-
```

Comment:
Let's continue to leave this in for now - now that there's a replacement, they have a version to replace the deprecated properties with the new options we're providing.